net-im/swiften/swiften-4.0.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit scons-utils toolchain-funcs
+
+DESCRIPTION="A C++ library for implementing XMPP applications"
+HOMEPAGE="https://www.swift.im/"
+SRC_URI="https://swift.im/downloads/releases/swift-${PV}/swift-${PV}.tar.gz"
+
+LICENSE="BSD BSD-1 CC-BY-3.0 GPL-3 OFL-1.1"
+SLOT="0/4"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug expat icu idn sqlite upnp zeroconf"
+
+DEPEND="dev-libs/boost:=
+	dev-libs/openssl:0=
+	sys-libs/zlib:=
+	expat? ( dev-libs/expat )
+	!expat? ( dev-libs/libxml2:2 )
+	icu? ( dev-libs/icu:= )
```

Comment:
did you want to remove it or did you forget to add it in MYSCONS?

Reply:
Hm, does in that case USE=icu makes still sense? I see the code, where it hides on IDN. Would it not be better, to drop ICU as a use flag an add icu depend on USE=idn?

Reply:
it appears to me that it is either libidn or icu (or both) for idn support

Reply:
But I don't see an option, to disable icu? So if someone sets IUSE="-icu", it won't disable linking, if maybe another package pulled in dev-libs/icu?

Reply:
yeah it will not be disabled in the current ebuild, it wold have to be handled.

there is `vars.Add(PackageVariable("icu", "ICU library location", "no"))` in SConscript.boot, so you'd probably have to pass its location if you want it, and "no" if you don't

Reply:
Seems to be the case. Objdump shows it. Commited.